src/Pact/Analyze/Translate.hs

```diff
@@ -628,8 +628,11 @@ translateNode astNode = withAstContext astNode $ case astNode of
     ESimple TStr nameT <- translateNode nameA
     return $ ESimple TDecimal $ ReadDecimal nameT
 
-  AST_ReadInteger _ -> throwError' $ UnexpectedNode astNode
-  AST_ReadMsg _     -> throwError' $ UnexpectedNode astNode
+  AST_ReadInteger nameA -> do
+    ESimple TStr nameT <- translateNode nameA
+    return $ ESimple TInt $ ReadInteger nameT
+
+  AST_ReadMsg _ -> throwError' $ UnexpectedNode astNode
```

Comment:
We should do this a la `NoPacts`, `NoLists`, etc. IE, this is not an internal error and should have a message saying we'll support this in the future.